File: Android/java/doraemonkit/src/main/java/com/didichuxing/doraemonkit/widget/easyrefresh/ILoadMoreView.java
package com.didichuxing.doraemonkit.widget.easyrefresh;

import android.view.View;

/**
 * Created by guanaj on 16/9/22.
 */

public interface ILoadMoreView {
    /**
     * 重置
     */
    void reset();

    /**
     * 加载中
     */
    void loading();

    /**
     * 加载完成
     */
    void loadComplete();

    void loadFail();

    void loadNothing();

    View getCanClickFailView();


}

File: Android/kotlin/doraemonkit/src/main/java/com/didichuxing/doraemonkit/widget/easyrefresh/ILoadMoreView.kt
package com.didichuxing.doraemonkit.widget.easyrefresh

import android.view.View

/**
 * Created by guanaj on 16/9/22.
 */
interface ILoadMoreView {
    /**
     * 重置
     */
    fun reset()

    /**
     * 加载中
     */
    fun loading()

    /**
     * 加载完成
     */
    fun loadComplete()
    fun loadFail()
    fun loadNothing()
    val canClickFailView: View
}